app/obfuscators/base64_no_padding.py

```diff
@@ -0,0 +1,35 @@
+from base64 import b64encode
+
+from app.utility.base_obfuscator import BaseObfuscator
+
+
+class Obfuscation(BaseObfuscator):
+
+    @property
+    def supported_platforms(self):
+        return dict(
+            windows=['psh'],
+            darwin=['sh'],
+            linux=['sh']
+        )
+
+    def __init__(self, agent):
```

Comment:
All the obfuscators have the same __init__ method -- what do you think about moving this into the BaseObfuscator class?

Reply:
Makes sense to me. Should this be a part of a new ticket?

Reply:
Or to put it in different terms, the base class should have the __init__ method, and the subclasses should only override it if they need to do something in addition to the base class.

Reply:
> Makes sense to me. Should this be a part of a new ticket?

I think it would be ok to include it as part of this ticket.